DuckDB.NET.Data/DuckDBCommand.cs

```diff
@@ -41,16 +41,16 @@ private int ExecuteScalarOrNonQuery()
         {
             var result = PlatformIndependentBindings.NativeMethods.DuckDBQuery(connection.NativeConnection, CommandText, out var queryResult);
 
-            if (!result.IsSuccess())
+            if (!string.IsNullOrEmpty(queryResult.ErrorMessage))
             {
-                throw new DuckDBException("DuckDBQuery failed", result);
+                throw new DuckDBException(queryResult.ErrorMessage);
```

Comment:
Can you pass the `result` to the constructor too?

Reply:
done.